draft-ietf-jsonpath-base.md

```diff
@@ -1440,7 +1443,7 @@ need escaping when Normalized Paths appear as strings (which are delimited with
 Certain characters are escaped, in one and only one way; all other characters are unescaped.
 
 Normalized Paths are Singular Paths. Not all Singular Paths are Normalized Paths: `$[-3]`, for example, is a Singular
-Path, but not a Normalized Path.
+Path, but not a Normalized Path, because **WHY (please explain why)?**.
```

Comment:
Since Normalized Paths are for expressing outputs, the array length is known and so a non-negative index can be used. We are trying to make sure there is only one way of writing a Normalized Path. This probably needs to explained earlier in this section.

In terms of this example, the basic reason `$[-3]` is not a Normalized Path is that it is not produced by the ABNF.

Reply:
I think it would be helpful for the language to contain that explanation.

Reply:
Ok, please see if 7293e20f135361b8f504fcaf1cef526e5265da1b is sufficient.